akiraux/Akira
# Display Artboard Sizes Panel when inserting Art-Board

## Summary / How this PR fixes the problem?

As discussed in #554, this PR implements the feature to display a list of common screen sizes to create a new artboard.

- Created a ArtboardSizesPanel class for displaying size categories in the left panel. 
- This component contains screen size categories like *Desktop*, *IPads* and *Mobiles*.
- The component also has a **+** button that the user can use to create new categories. 
- Each category will contain buttons, clicking on which will insert a artboard of the size it mentions.
- The categories have been implemented using Gtk.Expander.
- All the category names and their screen sizes will be saved in the *gschema.xml.in* file in the form of a JSON string.
- This panel will only be shown when inserting an artboard.
- The newly inserted artboard will be positioned intelligently based on the positions of other artboards

## Steps to Test

- Press ```A``` to insert artboard.
- In the left panel, you will see a bunch of categories.
- Click on the device category you want to expand it and reveal a bunch of screen sizes
- Clicking on any of these screen sizes will insert an art-board automatically. The canvas also zooms and scrolls to fit the new artboard.
- Click on the ```+``` button to create a new category.
- At the bottom of every category, a button is present for adding new screen sizes.

## Screenshots 
![artboard sizse pr](https://user-images.githubusercontent.com/64722310/129704430-2e1c6175-5431-42b4-b060-56503f939bdd.png)
